src/components/relationships.tsx

```diff
@@ -91,6 +157,48 @@ export const RelationshipsTab = (
           }
         />
       </Field>
+      {!queryBinder && (
+        <>
+          <Field label="Start ExtrenalId" className="relationships-select">
```

Comment:
```suggestion
          <Field label="Start ExternalId" className="relationships-select">
```
may I ask you to install spell checker?

Reply:
I use one but seems still not working (under "strings" at least)